docs/content/latest/secure/authentication/ldap-authentication-ycql.md

```diff
@@ -0,0 +1,235 @@
+   </td>
+  </tr>
+  <tr>
+   <td>ycql_ldap_server
+   </td>
+   <td>LDAP server endpoint of the form scheme://ip:port. Scheme can be ldap (or) ldaps.
+   </td>
+   <td>''
+   </td>
+  </tr>
+  <tr>
+   <td>ycql_ldap_tls (Not supported yet)
+   </td>
+   <td>Connect to LDAP server using TLS encryption
+   </td>
+   <td>false
+   </td>
+  </tr>
+</table>
+
+## Simple Bind Mode
+
+In **simple-bind** mode, YB-TServer will bind to the Distinguished Name (“DN”) constructed with “prefix username suffix” format. Here is an example for Simple bind mode:
+
+```sh
+--use_cassandra_authentication=true --ycql_use_ldap=true --ycql_ldap_server=ldap://ldap.yugabyte.com:389 --ycql_ldap_user_prefix=uid= --ycql_ldap_user_suffix=, ou=DBAs, dc=example, dc=com --ycql_ldap_users_to_skip_csv=cassandra
+```
+
+### Configurations
+
+The configuration specific to simple bind mode.
+
+<table>
+  <tr>
+   <td><strong>Tserver gflag name</strong>
+   </td>
+   <td><strong>Description</strong>
+   </td>
+   <td><strong>Default value</strong>
+   </td>
+  </tr>
+  <tr>
+   <td>ycql_ldap_user_prefix
+   </td>
+   <td>String used for prepending the user name when forming the DN for binding to the LDAP server
+   </td>
+   <td>''
+   </td>
+  </tr>
+  <tr>
+   <td>ycql_ldap_user_suffix
+   </td>
+   <td>String used for appending the user name when forming the DN for binding to the LDAP Server
+   </td>
+   <td>''
+   </td>
+  </tr>
+</table>
+
+## Search + Bind Mode
+
+In `Search + Bind` mode, YB-Tserver will bind to the LDAP directory with a fixed username and password, specified with `ycql_ldap_bind_dn` and `ycql_ldap_bind_passwd`, and performs a search for the user trying to log in to the database. This mode is commonly used by LDAP authentication schemes in other softwares.
+
+For Searching the LDAP directory if no fixed username and password is configured at YB-TServer, an anonymous bind will be attempted to the directory. The search will be performed over the subtree at `ycql_ldap_base_dn`, and will try to do an exact match of the attribute specified in `ycql_ldap_search_attribute`. Once the user has been found in this search, the server disconnects and re-binds to the directory as this user, using the password specified by the client, to verify that the login is correct.
+
+Here is an example for search + bind mode:
+
+```sh
+--use_cassandra_authentication=true --ycql_use_ldap=true --ycql_ldap_server=ldap://ldap.yugabyte.com:389 --ycql_ldap_base_dn="dc=yugabyte, dc=com" --ycql_ldap_bind_dn="cn=admin,dc=example,dc=org" --ycql_ldap_bind_passwd=admin --ycql_ldap_search_attribute=uid
+```
+
+### Configurations
+
+The configurations supported for search + bind mode.
+
+<table>
+  <tr>
+   <td><strong>Tserver gflag name</strong>
+   </td>
+   <td><strong>Description</strong>
+   </td>
+   <td><strong>Default value</strong>
+   </td>
+  </tr>
+  <tr>
+   <td>ycql_ldap_base_dn
+   </td>
+   <td>Specifies the base directory to begin the user name search
+   </td>
+   <td>''
```

Comment:
nit: maybe these empty strings (this an below) would look clearer as code snippets `''` vs ''.

Reply:
done